iqrm/core.py

```diff
@@ -93,6 +96,14 @@ def iqrm_mask(x, radius=5, threshold=3.0):
             votes_cast[j].add(i)
             votes_received[i].add(j)
 
+    # Ignore some channels
+    nchans = len(x)
+    for c in ignorechans:
+        other_chans = list(range(0,c)) + list(range(c+1, nchans))
```

Comment:
Please use `set` here because
1. The votes dicts are already dicts of sets
2. `set` has **much** faster lookups in the "vote tally" loop below, the speed difference might be dramatic for large nchans.

```
other_chans = set(range(0,c)) | set(range(c+1, nchans))
```